zend_abstract_interface/zai_sapi/zai_sapi.h

```diff
@@ -52,6 +56,9 @@ void zai_sapi_mshutdown(void);
 bool zai_sapi_rinit(void);
 void zai_sapi_rshutdown(void);
 
+/* Called from sapi_module_struct.register_server_variables */
+extern void (*register_custom_server_variables)(zval *track_vars_server_array TSRMLS_DC);
```

Comment:
One more nit (sorry!) Can we prefix this with `zai_sapi_` for consistency with the other ZAI SAPI APIs? :)